modules/dists/ReplicatedDist.chpl

```diff
@@ -424,6 +424,16 @@ class ReplicatedArr : BaseArr {
         halt("locale ", here.id, " has no local replicand");
     return localArrs[here.id];
   }
+
+  /* Access another locale's local array representation by locale id */
+  proc replicand(id: here.id.type) ref {
+    return localArrs[id].arrLocalRep;
+  }
+}
+
+pragma "no doc"
+proc _array.replicand(id : here.id.type) ref {
+  return _value.replicand(id);
```

Comment:
Once we can do forwarding as hoped on `_value` fields, we would expect this to go away, is that right?  If so, would you add a comment to that effect indicating that this is a placeholder until such time?  Or, is there any chance that Michael's commit of this morning already gets us to that point?